embulk-docs/src/release/release-0.9.12.rst

```diff
@@ -0,0 +1,13 @@
+Release 0.9.12
+==================================
+
+General Changes
+----------------
+
+* Display file name and file size in log to easier debug which is loaded by FileInputPlugin [#1066]
```

Comment:
#1066 shows only file name, don't show file size

Reply:
Ah, right. Thanks for the good catch! (I wrongly copied it from older git commit message.)